lib/src/common/variables.ts

```diff
@@ -219,6 +219,29 @@ export const componentTokens = {
     focusBorderThickness: CoreTokens.border_width_2,
     focusBorderRadius: CoreTokens.border_radius_medium,
   },
+  contextualMenu: {
+    fontFamily: CoreTokens.type_sans,
+    backgroundColor: CoreTokens.color_white,
+    borderColor: CoreTokens.color_grey_200,
+    menuItemFontColor: CoreTokens.color_grey_900,
+    menuItemFontSize: CoreTokens.type_scale_02,
+    menuItemFontStyle: CoreTokens.type_normal,
+    menuItemFontWeight: CoreTokens.type_regular,
+    menuItemLineHeight: "24px",
+    hoverMenuItemBackgroundColor: CoreTokens.color_grey_100,
+    activeMenuItemBackgroundColor: CoreTokens.color_grey_100,
+    selectedMenuItemBackgroundColor: CoreTokens.color_purple_100,
+    hoverSelectedMenuItemBackgroundColor: CoreTokens.color_purple_200,
+    activeSelectedMenuItemBackgroundColor: CoreTokens.color_purple_200,
+    selectedMenuItemFontWeight: CoreTokens.type_semibold,
+    sectionTitleFontColor: CoreTokens.color_grey_900,
+    sectionTitleFontSize: CoreTokens.type_scale_03,
+    sectionTitleFontStyle: CoreTokens.type_normal,
+    sectionTitleFontWeight: CoreTokens.type_semibold,
+    sectionTitleLineHeight: "24px",
+    iconColor: CoreTokens.color_grey_900,
+    iconSize: "16px",
```

Comment:
Should we consider `1rem` instead of `16px`

Reply:
I asked Lara she told me that we always use `px` for sizing icons, so we must keep it like that.